pkg/controller/endpointslicemirroring/endpointslice_tracker.go

```diff
@@ -45,61 +45,76 @@ func newEndpointSliceTracker() *endpointSliceTracker {
 	}
 }
 
-// has returns true if the endpointSliceTracker has a resource version for the
+// Has returns true if the endpointSliceTracker has a resource version for the
 // provided EndpointSlice.
-func (est *endpointSliceTracker) has(endpointSlice *discovery.EndpointSlice) bool {
+func (est *endpointSliceTracker) Has(endpointSlice *discovery.EndpointSlice) bool {
```

Comment:
nitpick: do these need to be exported prior to be moved into a shared package in https://github.com/kubernetes/kubernetes/pull/93443?

Reply:
Yeah you're completely right. I was updating this to match the EndpointSlice controller implementation precisely which already included exported functions. Since the end goal was to extract this to a shared package (https://github.com/kubernetes/kubernetes/pull/93443) it seemed like this would result in less long term conflicts. Especially if this commit ends up applying to earlier versions of Kubernetes than the shared package does. I'd thought this might be easier to backport than extracting the tracker into a shared package.